### PR TITLE
Yoni frontend

### PR DESCRIPTION
**Pull request by VinaiRachakonda**
-- imported yoni's code and configured it ts (some things are not properly typed)
-- pipeline view and company view integrated with localhost api (not actual api) 
-- some of this code is a little messy but I'm pushing just so we have some sort of content for us to start working with.

### CODE REVIEW

File: client/pages/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      description: elt['description']
+    };
+
+    /*
+     * If the company's status fits in one of the columns add it
+     * to that column.
+     */
+    if (elt['status'] in columns) {
+      columns[elt["status"]]['companyIds'].push(id);
+    }
```
Comment (casey-chow) [inline client/pages/utils.tsx:46]: If it doesn't we should have some kind of error right?

File: client/pages/PortfolioSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Layout from '../components/Layout/Layout';
+
+class PortfolioSuccesss extends React.Component {
```
Comment (casey-chow) [inline client/pages/PortfolioSuccess.tsx]: Transform to function component

**Review by casey-chow — APPROVED** [review]
1. Run `make lint-fix`--I see some formatting issues, I'll adjust the pre-commit to fix them.
2. Route component files should be lowercase per next.js convention.

Looks great otherwise.

File: client/pages/utils.tsx (hunk 1 above)
Comment (VinaiRachakonda) [reply client/pages/utils.tsx:46]: Yea good call. I should make the api call pull unarchvied companies as well.

File: client/pages/PortfolioSuccess.tsx (hunk 2 above)
Comment (VinaiRachakonda) [reply client/pages/PortfolioSuccess.tsx]: Good catch.

**Comment by VinaiRachakonda** [comment]
Make link-fix doesn't work for me.

**Pull request merged**